### PR TITLE
Use fmt.printf() to end with a newline

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
 fmt.printf() ends with a newline, Easy to read printed info.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/cmd/upgrade/common.go [hunk 1]
```diff
@@ -227,7 +227,7 @@ func getWaiter(dryRun bool, client clientset.Interface) apiclient.Waiter {
 // InteractivelyConfirmUpgrade asks the user whether they _really_ want to upgrade.
 func InteractivelyConfirmUpgrade(question string) error {
 
-	fmt.Printf("[upgrade/confirm] %s [y/N]: ", question)
+	fmt.Printf("[upgrade/confirm] %s [y/N]:\n", question)
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/upgrade/common.go]: we should omit this change, because the user should write the `y` or `n` on the same line as the prompt.

**Review by neolit123 — COMMENTED** [review]
added one minor comment.
other than that LGTM.

/priority backlog

File: cmd/kubeadm/app/cmd/upgrade/common.go (hunk 1 above)
Comment (xichengliudui) [reply cmd/kubeadm/app/cmd/upgrade/common.go]: I have neglected. It has been updated. Is there anything else that needs to be updated?

**Review by neolit123 — COMMENTED** [review]
thanks.
/lgtm
/approve

**Pull request merged**